### PR TITLE
Added HereSection and new arrivals section

### PR DESCRIPTION
**Pull request by Ankur9669**
Files to check:
1. App.js
2. HeroSection.js
3. NewArrivalsSection.js
4. NewArrivalsCard.js

The Css is copied from the previous project so you can omit reviewing the css.

# HeroSection
image

# NewArrivalsSection
image

### CODE REVIEW

**Review by Priya-Patel31 — APPROVED** [review]
Code looks clean to me and you can go ahead to merge those files.

File: fitbit-react/src/App.css [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
+:root{
+    --light-blue: #f2fbff;
+    --color-black: #000000;
+    --color-white: #ffffff;
+    --navbar-shadow: #b5b5b5;
+    --gray-color: #696969;
 }
-
-.App-link {
-  color: #61dafb;
```
Comment (bhtibrewal) [inline fitbit-react/src/App.css:28]: Suggestion: use variables for color

**Review by bhtibrewal — APPROVED** [review]
the code looks good to me, nice conventions used , okay to merge

**Review by varun1233 — APPROVED** [review]
Nice work. Code look good to merge

**Pull request merged**